ecosystem_cicd_tools/new_cicd/actions.py

```diff
@@ -1,7 +1,10 @@
 import os
+import time
 from copy import deepcopy
 from urllib.parse import urlparse
-
+from cfy_lint.yamllint_ext.utils import (get_node_types_for_plugin_version,
+                                         get_plugin_versions_from_marketplace,
+                                         get_plugin_id_from_marketplace)
```

Comment:
Don't use cfy_lint as an importable module. These functions are only examples how to use the API, they are not appropriate. Write new functions and put them in `ecosystem_cicd_tools/new_cicd/marketplace.py`. 

We need functions that:
- check the latest version of a plugin in the market place. Is it the expected `release_name`? (Unless `release_name` is "latest".)
- Check if all of the expected `assets` are in the plugin wagon/plugin YAML list in the marketplace.


Checking the node types would be nice, but I'm not sure it's practical.